simple/util/dc_client.py

```diff
@@ -17,26 +17,42 @@
 import os
 import requests
 from absl import logging
+import json
 
 from .ngram_matcher import NgramMatcher
 
-# Environment variable for API key.
+# Environment variables.
 _KEY_ENV = "DC_API_KEY"
+_API_ROOT_ENV = "DC_API_ROOT"
+
+# Default REST API endpoint root.
+_DEFAULT_API_ROOT = "https://api.datacommons.org"
 
 
 def get_api_key():
     return os.environ.get(_KEY_ENV, "")
 
 
-# REST API endpoint root
-_API_ROOT = "https://api.datacommons.org"
+def get_api_root():
+    return os.environ.get(_API_ROOT_ENV, _DEFAULT_API_ROOT)
+
+
+_DEBUG = True
+_DEBUG_FOLDER = ".data/debug"
```

Comment:
nit: move the constants to the top together.

Reply:
Done.